src/website/app/pages/FourOhFour/index.js

```diff
@@ -0,0 +1,97 @@
+/* @flow */
+import React from 'react';
+import { css } from 'glamor';
+import { createStyledComponent } from '../../../../styles';
+import Button from '../../SiteButton';
+import Link from '../../SiteLink';
+import Markdown from '../../Markdown';
+import content from './404.md';
+const floatingMineralsSvg = require('!!raw-loader!../../../public/images/minerals.svg');
+
+type Props = {};
+
+const random = (range, offset) => Math.floor(Math.random() * range) + offset;
+
+const getAnimation = (offset1, offset2) => {
+  let frames = {};
+  for (let i = 10; i <= 100; i += 10) {
+    frames[`${i}%`] = {
+      clipPath: `inset(${random(25, offset1)}% 0 ${random(25, offset2)}% 0)`
+    };
+  }
+  return frames;
+};
+
+const glitch1 = css.keyframes('glitch1', getAnimation(0, 40));
+const glitch2 = css.keyframes('glitch2', getAnimation(40, 5));
+
+const mineralCopyStyles = {
+  backgroundColor: 'white',
+  position: 'absolute',
+  top: 0,
+  width: '100%'
+};
+
+const styles = {
+  floatingMineralsContainer: {
+    filter: 'hue-rotate(150deg) contrast(150%)',
+    margin: '1em auto',
+    maxWidth: '25em',
+    overflow: 'hidden',
+    position: 'relative',
+    transform: 'rotate(180deg)',
```

Comment:
My opinion is that leaving the image upright is better.  I did not even recognize what it was when flipped.